### ISSUE TITLE
Rethink about the `core` library placement in Docs.Ms

### ISSUE THREAD

**Issue by sima-zhu** [Azure/azure-sdk#5345]
Currently, we put `Core` library under `Others` section in Docs.Ms as team thought `core` library is not the same concept as other azure services, so we shouldn't put it in parallel with other services.
<img width="217" alt="image" src="https://user-images.githubusercontent.com/48036328/211928206-68059dc9-de6d-45ad-b648-471f7b7a73ed.png">

However, @JonathanGiles from Java team prefer to have core back to azure service list but remove any `Service` header.

**Comment by JonathanGiles** [comment]
Yeah, to me the core APIs are critical for people to know about, but we lower their priority to 'other'. It would seem we've constrained ourselves to a service centric view, just because we have a 'services' header on a page. I would suggest we consider renaming that to 'Libraries' or similar, and elevate our core libraries up into core. Similarly, we put the core libraries under 'Other' on this page too: https://azure.github.io/azure-sdk-for-java/other.html

**Comment by weshaggard** [comment]
@scottaddie @lfraleigh @Petermarcu This was discussed many times and we decided to group them under "Other" as the docs site is about Azure not these helper libraries. If we want to change that we should get PM onboard and make the change consistent across the board.

**Comment by lfraleigh** [comment]
Thanks @weshaggard. We've got an email thread going on this too (no decision yet).

**Comment by kurtzeborn** [comment]
Pretty sure this conversation has run it's course.  If we're not in a good place now, please open a new issue with the desired outcome.
